pandas/core/arrays/datetimes.py

```diff
@@ -261,6 +261,7 @@ def _scalar_type(self) -> type[Timestamp]:
     )
     _datetimelike_methods: list[str] = [
         "to_period",
+        "to_timestamp",
```

Comment:
This method should not be added to `DatetimeArray`; it already exists on `PeriodArray`.

However you do want a different docstring for `Series.dt.to_datetime` than the one on `PeriodArray`. I think this is a systemic issue that impacts all `dt` methods. We can use `ft.wraps` as you are doing here, but then overwrite the docstring. We will need a place to put the docstrings, and then write one for each method in the `dt` namespace. This seems to me to be a large effort, and should not be undertaken before being thoroughly discussed.

If there are better, low effort, solutions than the above, I'm certainly willing to entertain them here. But if not, I think we can move forward with this PR after removing the changes to this file.

Reply:
Thank you for the review. I agree that the `dt` methods need separate handling as you have suggested. Will revert these changes along with the other suggested edits for this PR.